src/shared/extension-release.c

```diff
@@ -77,3 +77,18 @@ int extension_release_validate(
         log_debug("Version info of extension '%s' matches host.", name);
         return 1;
 }
+
+int parse_env_extension_hierarchies(char ***ret_hierarchies) {
+        int r;
+
+        r = getenv_path_list("SYSTEMD_SYSEXT_HIERARCHIES", ret_hierarchies);
+        if (r < 0)
+                return log_debug_errno(r, "Failed to parse SYSTEMD_SYSEXT_HIERARCHIES environment variable : %m");
```

Comment:
Hmm, we generally ignore invalid variable values. Especially if this is used in "shared" code. Shouldn't the error be suppressed here?

Reply:
it was copied from the existing implementation verbatim - but you are right, I'll send a follow-up

Reply:
https://github.com/systemd/systemd/pull/18796